pep-0582.rst

```diff
@@ -10,7 +10,7 @@ Type: Standards Track
 Topic: Packaging
 Content-Type: text/x-rst
 Created: 16-May-2018
-Python-Version: 3.8
+Python-Version: 3.12
```

Comment:
Could you add the Post-History header with dates linking to PRs (or discussions, I don’t remember the process details)?

Reply:
Yeah its discussions; as far as I can tell, its all in the one big thread listed in `Discussions-To`, so it would be

```suggestion
Python-Version: 3.12
Post-History: `28-Feb-2019 <https://discuss.python.org/t/pep-582-python-local-packages-directory/963/173>`__
```

Given how long its gotten, it might be worth creating a new one at an appropriate breakpoint and cross-linking the old one.